app/models/item.rb

```diff
@@ -47,4 +47,18 @@ def updates_active
       update(active?: true)
     end
   end
+
+  def fulfill_item(order)
+    item_orders.each do |item_order|
+      if item_order.order == order
+        item_order.update(status: "fulfilled")
+      end
```

Comment:
Add item model method that will fulfill only items in an order for a certain item and order